src/main/webapp/app/localci/build-queue/build-queue.component.html

```diff
@@ -198,20 +281,57 @@ <h3 id="build-queue-queued-heading" jhiTranslate="artemisApp.buildQueue.queuedBu
                 </ngx-datatable-column>
                 <ngx-datatable-column prop="buildConfig.commitHash" [minWidth]="150" [width]="200">
                     <ng-template ngx-datatable-header-template>
-                        <span class="datatable-header-cell-wrapper" (click)="controls.onSort('commitHash')">
+                        <span class="datatable-header-cell-wrapper" (click)="controls.onSort('buildConfig.commitHash')">
                             <span class="datatable-header-cell-label bold sortable" jhiTranslate="artemisApp.buildQueue.buildJob.commitHash"></span>
-                            <fa-icon [icon]="controls.iconForSortPropField('commitHash')" />
+                            <fa-icon [icon]="controls.iconForSortPropField('buildConfig.commitHash')" />
                         </span>
                     </ng-template>
-                    <ng-template ngx-datatable-cell-template let-value="value">
-                        <span class="wrap-long-text">{{ value }}</span>
+                    <ng-template ngx-datatable-cell-template let-value="value" let-row="row">
+                        @if (
+                            row.repositoryInfo.triggeredByPushTo === 'TEMPLATE' ||
+                            row.repositoryInfo.triggeredByPushTo === 'SOLUTION' ||
+                            row.repositoryInfo.triggeredByPushTo === 'TESTS'
```

Comment:
Since you use these values quite often, we could maybe use an enum for that? (maybe in a future PR)